docs/architecture/cloud-native/defining-cloud-native.md

```diff
@@ -0,0 +1,335 @@
+---
+# Defining cloud native
+
+Stop what you’re doing and text 10 of your colleagues. Ask them to define the term “Cloud Native.” Good chance you’ll get eight different answers. Interestingly, six months from now, as cloud-native technologies and practices evolve, so will their definition.
+
+Cloud native is all about changing the way we think about constructing critical business systems.
+
+Cloud-native systems are designed to embrace rapid change, large scale, and resilience. 
+
+The Cloud Native Computing Foundation provides an [official definition](https://github.com/cncf/foundation/blob/master/charter.md):
+
+> *Cloud-native technologies empower organizations to build and run scalable applications in modern, dynamic environments such as public, private, and hybrid clouds. Containers, service meshes, microservices, immutable infrastructure, and declarative APIs exemplify this approach.*
+
+> *These techniques enable loosely coupled systems that are resilient, manageable, and observable. Combined with robust automation, they allow engineers to make high-impact changes frequently and predictably with minimal toil.*
+
+Applications have become increasingly complex with users demanding more and more. Users expect rapid responsiveness, innovative features, and zero downtime. Performance problems, recurring errors, and the inability to move fast are no longer acceptable. They'll easily move to your competitor.
+
+Cloud native is much about *speed* and *agility*. Business systems are evolving from enabling business capabilities to weapons of strategic transformation, accelerating business velocity and growth. It’s imperative to get ideas to market immediately. 
+
+Here are some companies who have implemented these techniques. Think about the speed, agility, and scalability they've achieved.
+
+| Company | Experience | 
+| :-------- | :-------- |
+| [Netflix](https://www.infoq.com/news/2013/06/netflix/) | Has 600+ services in production. Deploys a hundred times per day. |
+| [Uber](https://eng.uber.com/micro-deploy/) | Has 1,000+ services stored in production. Deploys several thousand builds each week. | WeChat | Has 300+ services in production. Makes almost 1,000 changes per day. |
+| [WeChat](https://www.cs.columbia.edu/~ruigu/papers/socc18-final100.pdf) | Has 300+ services in production. Makes almost 1,000 changes per day. |
+
+As you can see, Netflix, Uber, and WeChat expose systems that consist of hundreds of independent microservices. This architectural style enables them to rapidly respond to market conditions. They can instantaneously update small areas of a live, complex application, and individually scale those areas as needed.
+
+The speed and agility of cloud native come about from a number of factors. Foremost is cloud infrastructure. Five additional foundational pillars shown in Figure 1-1 also provide the bedrock for cloud-native systems.
+
+![Cloud native foundational pillars](media/cloud-native-foundational-pillars.png)
```

Comment:
The media files are not following the correct pattern. Each media should be inside a folder that matches the article name. More info at https://github.com/dotnet/docs/blob/master/CONTRIBUTING.md#example-structure. 

```suggestion
![Cloud native foundational pillars](./media/cloud-native-foundational-pillars.png)
```

Reply:
Confused.  Every image for the book lives in the single /media folder in the root. What is the purpose of the period? Not sure what you're saying here.

Reply:
The period is just how we typically reference subfolders

Reply:
Discussed. Maira will update in Ch 1. I will update in other chapters.